src/sdk/namespace_fs.js

```diff
@@ -496,15 +495,22 @@ class NamespaceFS {
     }
 
     async _upload_stream(source_stream, upload_path, write_options) {
-        return new Promise((resolve, reject) =>
-            source_stream
-            .once('error', reject)
-            .pipe(
-                fs.createWriteStream(upload_path, write_options)
-                .once('error', reject)
-                .once('finish', resolve)
-            )
-        );
+        let target_file;
+        try {
+            target_file = await nb_native().fs.open(DEFAULT_FS_CONFIG, upload_path, 'a');
```

Comment:
Why "a" ??? we do not want to keep appending to existing file in any case.
I would open as "w" or "wx".